src/Libraries/DynamoWatch3D/Watch3D.csproj

```diff
@@ -44,19 +44,19 @@
   <ItemGroup>
     <Reference Include="Cyotek.Drawing.BitmapFont, Version=1.0.2.0, Culture=neutral, PublicKeyToken=58daa28b0b2de221, processorArchitecture=MSIL">
       <HintPath>..\..\packages\Cyotek.Drawing.BitmapFont.1.0.2.0\lib\net20\Cyotek.Drawing.BitmapFont.dll</HintPath>
-      <Private>False</Private>
+      <Private>True</Private>
```

Comment:
Same as above, when you bring in packages, the dlls get set to copy local. I've got a cleanup target that removes extraneous SharpDX dlls from the nodes folder.